docs/tutorials/learning.rst

```diff
@@ -0,0 +1,151 @@
+cloning the repository:
+
+.. code-block:: console
+
+    git clone https://github.com/Qiskit/qiskit-experiments.git
+    python -m pip install -e qiskit-experiments
+
+The ``-e`` option will keep your installed package up to date as you make or pull new 
+changes.
+
+Running your first experiment
+=============================
+
+Let's run a T1 experiment. First, we have to import the T1 experiment from the 
+Qiskit Experiments library:
+
+.. jupyter-execute::
+
+    from qiskit_experiments.library import T1
+
+Experiments must be run on a backend. We're going to use a simulator, 
+:class:`qiskit.providers.fake_provider.FakeVigo`, for 
+this example, but you can use any IBM backend that you can access through Qiskit.
+
+.. jupyter-execute::
+
+    from qiskit.providers.fake_provider import FakeVigo
+    from qiskit_aer import AerSimulator
+    from qiskit.providers.aer.noise import NoiseModel
+    import numpy as np
+
+    # Create a pure relaxation noise model for AerSimulator
+    noise_model = NoiseModel.from_backend(
+        FakeVigo(), thermal_relaxation=True, gate_error=False, readout_error=False
+    )
+
+    backend = AerSimulator.from_backend(FakeVigo(), noise_model=noise_model)
+    qubit0_t1 = backend.properties().t1(0)
+
+    delays = np.arange(1e-6, 3 * qubit0_t1, 3e-5)
+    exp = T1(qubit=0, delays=delays)
+    exp_data = exp.run(backend=backend, seed_simulator=101).block_for_results()
+
+Run and display results:
+
+.. jupyter-execute::
+
+    exp_data = exp.run(backend=backend, seed_simulator=101).block_for_results()
+
+    # Print the result
+    display(exp_data.figure(0))
+    for result in exp_data.analysis_results():
+        print(result)
+
+
+Setting experiment options
+==========================
+
+Often it's insufficient to run an experiment with only the default options. 
+There are four types of options one can set for an experiment:
+
+* **Run options**, for passing to the experiment's ``run()`` method. Any run option that 
+  Terra supports can be set and will be passed to the jobs at run time:
+
+.. code-block::
+
+  exp.set_run_options(meas_level=MeasLevel.CLASSIFIED)
+
+* **Transpile options**, for passing to the Terra transpiler:
+
+.. code-block::
+
+  exp.set_transpile_options(scheduling_method='asap')
+
+* **Experiment options**, unique to the experiment class. Consult the API references 
+  for the options for each experiment.
+  
+.. code-block::
+
+  exp.set_experiment_options(delays=fields["delays_t1"])
+
+* **Analysis options**, unique to the analysis class. Consult the API references for the
+  options for each experiment analysis. Unlike the other options, this one is
+  not set via the experiment object but via analysis instead:
+
+.. code-block::
+
+  exp.analysis.set_options(gate_error_ratio=None)
+
+
+Running experiments on multiple qubits
+======================================
+
+To run experiments across many qubits of the same device, we use **composite experiments**.
```

Comment:
Batch experiments should be described separately, and not in the context of running on multiple qubits.